src/creature.cpp

```diff
@@ -158,6 +158,11 @@ tripoint Creature::pos() const
     return get_map().getlocal( location );
 }
 
+tripoint_abs_ms Creature::global_pos() const
+{
+    return location;
```

Comment:
There already exists a function that does exactly this: `Creature::get_location`.